lib/util.js

```diff
@@ -20,8 +20,8 @@ You can install it by running:
 }
 
 function * ensureNonStarterPlan (db) {
```

Comment:
😬 can we change the method name too? `ensureEssentialTierPlan` ?

Reply:
May as well.